### PR TITLE
Added coverage test 0518

### PR DESCRIPTION
**Pull request by Vihanakangas**

### CODE REVIEW

File: coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+#     vec4 tmp;
+#     // Always true.
+#     if(gl_FragCoord.x > _float_0_0)
+#     {
+#         tmp = _GLF_color;
+#     }
+#     _GLF_color = tmp;
+# }
+#
+# int func1()
```
Comment (afd) [inline coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber:75]: Comment that this function is guaranteed to return 2.

File: coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+# };
+# // Contents of three: 3
+# layout(set = 0, binding = 3) uniform buf3
+# {
+#     int three;
+# };
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# void func0()
```
Comment (afd) [inline coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber:64]: Comment that this function is a no-op, as it backs up and then restores _GLF_color.

File: coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+#     {
+#         _GLF_color = unpackSnorm4x8(_uint_60433);
+#     }
+#
+#     int a = _int_0;
+#     for(int i = 0; i < 5; i ++)
+#     {
+#         for(int j = 0; j < 2; j ++)
+#         {
+#             a += func1();
```
Comment (afd) [inline coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber:113]: Comment that this is his 10 times, and that func1() always returns 2, hence a ends up with the value 20.

**Review by afd — CHANGES_REQUESTED** [review]

**Comment by Vihanakangas** [comment]
Updated.

**Review by afd — APPROVED** [review]

**Pull request merged**